src/main/java/ca/corefacility/bioinformatics/irida/service/SequencingObjectService.java

```diff
@@ -140,7 +141,8 @@
 	 *                        the sample
 	 * @return the new {@link SampleSequencingObjectJoin}
 	 * @throws ConcatenateException if there was an error concatenating the sequences
+	 * @throws IOException if there was an error reading the sequences to concatenate
 	 */
 	public SampleSequencingObjectJoin concatenateSequences(List<SequencingObject> toJoin, String filename,
-			Sample targetSample, boolean removeOriginals) throws ConcatenateException;
+			Sample targetSample, boolean removeOriginals) throws ConcatenateException, IOException;
```

Comment:
This doesn't actually throw `IOException` anymore.

Reply:
Removed in 56ac519f9ce8049dfaf894978357291373d24e8a